### PR TITLE
Make QR Code visible on small screens (fixes #1602)

### LINKED ISSUE #1602: QR Code not fully visible on small Android screen
And when touch screen is used to move QR code to be fully visible, it's the background page which moves, and not the overlay with the QR code.

This was using the web interface (Bootstrap) on an older small Android phone. On a modern phablet, it works well.

2015-04-07-syncthing-qrcode-small-screen

### PR DESCRIPTION
**Pull request by bigbear2nd**
Added another line in the menu on small devices. Maybe not the most elegant solution.

### CODE REVIEW

File: gui/index.html [hunk 1]
```diff
@@ -50,6 +50,7 @@
           <ul class="dropdown-menu">
             <li><a href="" ng-click="editSettings()"><span class="glyphicon glyphicon-cog"></span>&emsp;<span translate>Settings</span></a></li>
             <li><a href="" ng-click="idDevice()"><span class="glyphicon glyphicon-qrcode"></span>&emsp;<span translate>Show ID</span></a></li>
+            <li><a href="" class="visible-xs" ng-click="qrDevice()"><span class="glyphicon glyphicon-qrcode"></span>&emsp;<span translate>Show QR-Code</span></a></li>
```
Comment (calmh) [inline gui/index.html]: I think the correct capitalization etc here is "QR code"

**Comment by calmh** [comment]
OK, so this works, but why is it necessary? :) Don't our other modals work on small screens, i.e. the settings, folder and device modals surely require scrolling to work?

**Comment by wweich** [comment]
The Android app does not seem to have the option to show the QR code (only by sharing it e.g. with barcode scanner).
Therefor a user has to use the WebUI. The WebUI inside the app uses the WebView, which seems to be somewhat broken in pre Kitkat versions which causes an inability to scroll the overlays.

I think a better fix would be to add the option to show the QR code inside the Android app, than adding this option to the main program.

**Comment by calmh** [comment]
Fwiw the scrolling works fine in Safari on iOS. So as you say, I think this is a webview bug and I'm not opposed to work around / fix it if we can, but I don't think this is the right solution.

**Comment by calmh** [comment]
Closing in waiting for some cleaner way to solve this, if it in fact still needs solving.

**Pull request closed without merging**